modules/apps/commerce/commerce-product-type-grouped-web/src/main/resources/META-INF/resources/view_cp_definition_grouped_entries.jsp

```diff
@@ -210,39 +214,40 @@ renderResponse.setTitle(cpDefinition.getName(themeDisplay.getLanguageId()));
 	}
 </aui:script>
 
-<aui:script use="liferay-item-selector-dialog">
-	window.document
-		.querySelector('#<portlet:namespace />addDefinitionGroupedEntry')
-		.addEventListener('click', (event) => {
-			event.preventDefault();
+<aui:script sandbox="<%= true %>">
+	const addDefinitionGroupedEntryButton = document.getElementById('<portlet:namespace />addDefinitionGroupedEntry');
 
-			var itemSelectorDialog = new A.LiferayItemSelectorDialog({
-				eventName: 'productDefinitionsSelectItem',
-				on: {
-					selectedItemChange: function (event) {
-						var <portlet:namespace />addCPDefinitionIds = [];
+	if (addDefinitionGroupedEntryButton) {
+		addDefinitionGroupedEntryButton.addEventListener('click', (event) => {
+			event.preventDefault();
 
-						var selectedItems = event.newVal;
+			Liferay.Util.openSelectionModal({
+				multiple: true,
+				onSelect: function (selectedItems) {
+					if (selectedItems?.length) {
```

Comment:
We cannot use optional chaining in JSPs, unfortunately. That is what source formatter is complaining about. Here is some strange unknown guy with more details: https://github.com/liferay-frontend/liferay-portal/pull/1015#issuecomment-832767394

Reply:
👍